desktop_files/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
That's fine because https://github.com/status-im/status-react/blob/develop/.babelrc is a one liner but maybe it would be better to have a base `.babelrc` which could be extended by each platform specific `.babelrc`?